schema/config/account.json

```diff
@@ -12,6 +12,41 @@
         "lifetime": {
             "$ref": "time-offset.json"
         },
+        "loginCookie": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "settings": {
+                    "description": "Security settings for login cookies",
+                    "type": "object",
+                    "properties": {
+                        "domain": {
+                            "type": "string"
+                        },
+                        "httpOnly": {
+                            "type": "boolean"
+                        },
+                        "maxAge": {
+                            "description": "Cookie lifetime in seconds",
+                            "type": "integer"
+                        },
+                        "path": {
+                            "type": "string"
+                        },
+                        "secure": {
```

Comment:
I think that the answer is "no" because it isn't required that the cookie is locked down to a specific path nor that the OpenToken service runs over HTTPS.  Because they are optional configuration, I don't require them in the schema.